Stage2 l1 matching

gpetruc opened the pull request:
I create it as a dummy PR here on my github so I can make some line-by-line comments

thomreis committed:
configuration file for uGMT lut dumper module

bortigno committed:
First compiling version

bortigno committed:
moving from label to token

grzanka committed:
Added CTPPS and TOTEM detector ids

grzanka committed:
Reduced number of detectors to 1, replaced TOTEM and CTPSS with VeryForward.

jan-kaspar committed:
Import from CTPPS repository: data formats for raw-to-digi conversion.

jan-kaspar committed:
Adaptation to upstream change of DetId.

jan-kaspar committed:
Import from CTPPS repository: conditions formats needed for TOTEM RP
raw-to-digi.

mulhearn committed:
third step in migration from static variables in OMTF configuration

Conflicts:
	L1Trigger/L1TMuonOverlap/interface/AngleConverter.h
	L1Trigger/L1TMuonOverlap/interface/OMTFConfiguration.h
	L1Trigger/L1TMuonOverlap/plugins/OMTFPatternMaker.cc
	L1Trigger/L1TMuonOverlap/src/AngleConverter.cc
	L1Trigger/L1TMuonOverlap/src/GhostBuster.cc
	L1Trigger/L1TMuonOverlap/src/GoldenPattern.cc
	L1Trigger/L1TMuonOverlap/src/OMTFConfigMaker.cc
	L1Trigger/L1TMuonOverlap/src/OMTFConfiguration.cc
	L1Trigger/L1TMuonOverlap/src/OMTFProcessor.cc
	L1Trigger/L1TMuonOverlap/src/OMTFReconstruction.cc
	L1Trigger/L1TMuonOverlap/src/OMTFResult.cc
	L1Trigger/L1TMuonOverlap/src/OMTFSorter.cc
	L1Trigger/L1TMuonOverlap/src/OMTFinput.cc
	L1Trigger/L1TMuonOverlap/src/OMTFinputMaker.cc
	L1Trigger/L1TMuonOverlap/src/XMLConfigReader.cc
	L1Trigger/L1TMuonOverlap/src/XMLConfigWriter.cc

jan-kaspar committed:
Implemented suggestions by Slava.

akalinow committed:
Final import of modifiactions from 2df598b48ab7a4cad0b132edb65897f14c6a0cad

akalinow committed:
* initial commit fixing OMTF configuration. Does not compile.

Commit:
pfMet Type-1 correction applied and the addition of caloMet in the sums branch of ntuples

akalinow committed:
* use two instances of L1TMuonOverlapParams: one for slow changing firmawre configurations,
  other for faster changing patterns data.

akalinow committed:
* continue removing global access to configuration parameters. Not yet finished.

akalinow committed:
* final removal of OMTFConfiguration::instance() use.

akalinow committed:
* final commit of OMTFConfiguration cleaning

akalinow committed:
* cleanup cfg.py files
* remove split of the OMTF configuration into two DB records (maybe put back if needed)

jan-kaspar committed:
TotemRawData merged into TotemDigi, L1TotemRP removed.

jan-kaspar committed:
Update after modifying raw-data DataFormats.

jimbrooke committed:
add gen tree

jan-kaspar committed:
Added default constructor to TotemTriggerCounters.

thomreis committed:
Construct MicroGMTLUT from l1t::LUT

thomreis committed:
Possibility to construct AbsoluteIsolationCheckLUT from l1t::LUT

thomreis committed:
MicroGMTLUT creation from l1t::LUT for the rest of the uGMT LUT types

thomreis committed:
Use LUTs from conditions instead of from files

jan-kaspar committed:
TotemTriggerCounters: reverting to struct.

smuzaffar committed:
Fix for runtestTqafTopEventSelection unit test: use againstElectronVLooseMVA6 instead of againstElectronVLooseMVA5

(cherry picked from commit 7b86dbde8a2f95f184dd5a7e6f5ad21fa9e28063)

jimbrooke committed:
add generator info tree

jimbrooke committed:
change HCAL TP scale to CaloTPGTranscoder

jimbrooke committed:
remove legacy L1Tree and L1MenuTree, fix naming of l1uGTTrees

jimbrooke committed:
add missing files for MC

jimbrooke committed:
bugfixes and naming corrections for gen info in ntuples

jan-kaspar committed:
Initial import from the CTPPS repository.

l-cadamuro committed:
no secondaries left-right trimming

gflouris committed:
phib st2 set to zero

gflouris committed:
setfinebit false

akalinow committed:
* fix a transparent bug assigning -1 to unsigned int.
  (it was transparent, as resuiltig value was assigne to int,
   resulting back in -1)
* set missing HwSignValid to always true

abrinke1 committed:
LUT tables from mrcarver PUFix commit, Finally have correct xmls ...

tstreble committed:
Added charge for offline electrons in L1TNtuples

abrinke1 committed:
Fixes to PrimitiveConverter, run on csctfDigis for data

Commit:
Added caloMetBE, caloMetPhiBE and caloSumEtBE to the ntuples

Commit:
Fixed syntax error

abrinke1 committed:
Crucial fix to typo in PtAssignment.h, dPhi = fabs(dPhi)

l-cadamuro committed:
fix saturation

tstreble committed:
Modified EG emulator to match firmware implementation

Commit:
Bugfix so that caloMetBE /= caloMet

tstreble committed:
Updated EG iso

Updated EG iso to deal with tower compression feature

mrcarver committed:
added emtf geometry generator for Alex

mrcarver committed:
Added fix to how track eta is assigned for pT assignment

mrcarver committed:
Fixed typo in charge assignment script

bundocka committed:
calo tools caloEta fix

rekovic committed:
Update customise: For 2016 L1TReEmul(MC)FromRAW from unpacked TPs, use Bmtf unpacker; For 2015  Customise L1TReEmul(MC)FromRAW2015 from L1T input Digis, except for HcalTP.

thomreis committed:
handle wrap-around for negative globPhi values since EMTF hwPhi can be as low as -35

rekovic committed:
Update testL1T to use customise=L1TReEmulromRAW2015.

mulhearn committed:
bugfixes for ntupling 2016 RAW data, and add integration test of same

mulhearn committed:
bugfixes for ntupling 2016 RAW data, and add integration test of same

l-cadamuro committed:
hasEM only for seed TT

rekovic committed:
L1Ntuple: add l1UpgradeTfMuonTree to EMU and RAW workflows.

Commit:
Implemented trig function lookup in calculation of Et/Ht sums

l-cadamuro committed:
iso window size

Commit:
Implemented temp fix to ensure alignment for demux jet iphi

Commit:
Increased the number of bins in Demux Jet iphi histogram

rekovic committed:
Remove HCALTP configuration from L1TCalorimeter simulation configuration. It belongs (and is) in L1Trigger.Configuration.CaloTriggerPrimitives_cff and should not be called in multiple locations.

rekovic committed:
Remove HCALTP configuration from customisation HFTP1x1. Called in L1Trigger.Configuration.CaloTriggerPrimitives_cff when ReEmul. Add simEcal/HcalTP to 2016 ReEmul

rekovic committed:
Keep call to HCALTP configuration in L1Ntuple CaloTowerTree configuration, but do not change LUTGenerationMode.

rekovic committed:
Pick up simEcalTriggerPrimitives to be filled in CaloTowerEmuTree

Commit:
Implemented changes to range of quantities used in calculation of x,y components of MET and MHT in accordance to latest firmware

rekovic committed:
Add customise L1TReEmulFromRAWsimEcalTP workflow for 2016 data using simEcalTPs and HcalTP digis

rekovic committed:
Augment unit test with RAW 2016 data consistency compare2 test.

dinyar committed:
Improvements to cancel-out units.
Increased possible dEta window to 0.31, allowed use of parameters fPhi and fEta to make cancel-out window oblong, and layed groundwork to use eta fine bit.

Conflicts:
	L1Trigger/L1TMuon/plugins/L1TMuonGlobalParamsESProducer.cc
	L1Trigger/L1TMuon/src/MicroGMTCancelOutUnit.cc
	L1Trigger/L1TMuon/src/MicroGMTMatchQualLUT.cc

thomreis committed:
Increase cancel out LUT size to 5 bit dEta (Total with increases to 8 bit)

Commit:
Moved trig lookup list for MET/MHT into CaloTools

thomreis committed:
Fix uGMT emulator but that too many muons pass the sorter

mulhearn committed:
update EMTF LUT data file path to as pushed to external

mulhearn committed:
fix bad merge of l1JetRecoTree cfi

mulhearn committed:
post rebase cleanup of caloStage2Params

mulhearn committed:
fix bad merge

mulhearn committed:
post rebase fix of OMTFPatternMaker

mulhearn committed:
remove config files with multiple dots, which confuse parser

rekovic committed:
Unit test: use collision runs when testing unpakcing

rekovic committed:
Configuration: Add re-emulation workflows L1TReEmulFromRAWsimTP, L1TReEmulFromRAW2015simCaloTP.

mulhearn committed:
manually sync calo stage2 params (non-default) file to match v42.4, poorly rebased due to name changes

smorovic committed:
mergeType PSet parsing and new JSON field. untracked parameter mergeTypePset specifies which pSet name to use

mulhearn committed:
revert writing of new EMTF output formats from EndCap Track Producer

Conflicts:
	L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapTrackProducer.cc

mulhearn committed:
post rebase manual sync of L1TMuonEndCap

mulhearn committed:
post rebase cleanup of L1TNuples

nsmith- committed:
Fix some connections for Stage1 trigger emulation from RAW

abrinke1 committed:
Modified EMTF unpacker to output CSCCorrelatedLCTDigiCollection

abrinke1 committed:
Revert to proper default for simEmtfDigis_cfi.py

blwiner committed:
Shift uGT Readout Word definitions

bartosik-hep committed:
Adjusted HLT path/module names to the ones from /online/collisions/2016/25ns10e33/v1.0/HLT/V4 configuration

alja committed:
Implement palettes.

bartosik-hep committed:
Removed redundant version numbers from trigger-path names

dinyar committed:
Preliminary cancel-out parameters.

dinyar committed:
Adding Thomas' fixes.

dinyar committed:
Small fixes.

eparadas committed:
Adding BMTF unpacker files

eparadas committed:
removing debug cout statements

davignon committed:
Sorting with descending pT of eg and tau candidates at Demux

tstreble committed:
Fixed accidental commit affecting the phi position of L1Calo objects

nsmith- committed:
Adding new CaloLayer1 calibrations for upstream calibration studies

Changes:
1) Re-fix endcap = 1 discontinuity
2) New HF scale factors, as shown to L1-DPG Apr. 26 2016

rekovic committed:
Redefine L1TReEmulFromRAW (2016 data): simBmtfDigis use unpacked payloads of Bmtf instead of simTwinMux

davignon committed:
Adapt Tau ID flags for 2016 Reco

rekovic committed:
EMTF: fix shorthanded if-else statement for ranking in PatternRecognition

mulhearn committed:
Minimal required cleanup of EMTF Pt assigment, more coming

thomreis committed:
Including the eta fine bit in the cancel out LUT. fwVersion selects LUT with or without eta fine support.

thomreis committed:
Optimised cancel out window parameters from Dinyar

mulhearn committed:
unpacker cleanup requested by David L

mulhearn committed:
code cleanup for PR 14364 at request of David L

gpetruc commented:
@battibass

gpetruc commented on MuonAnalysis/MuonAssociators/plugins/L1MuonMatcher.cc:
```diff
@@ -95,23 +107,49 @@ pat::L1MuonMatcher::produce(edm::Event & iEvent, const edm::EventSetup & iSetup)
+      for (int ibx = l1tBX->getFirstBX(); ibx <= l1tBX->getLastBX(); ++ibx)
+	bxIdxs.push_back(l1tBX->end(ibx) - l1tBX->begin());
+    }
+    else {
```
secondo me sarebbe piu' leggibile mettendo le graffe attorno al for, e mettere la graffa prima dell'else alla stessa riga dell'else, altrimenti potrebbe sembrare che la graffa a 132 chiuda il for

battibass replied:
Corretto, scusami e' che di solito scrivo + largo (mandando a capo tutte le graffe) e avevo compattato secondo il tuo stile, ma l'ho fatto male .

gpetruc commented on MuonAnalysis/MuonAssociators/plugins/L1MuonMatcher.cc:
```diff
@@ -120,21 +158,50 @@ pat::L1MuonMatcher::produce(edm::Event & iEvent, const edm::EventSetup & iSetup)
+	    int charge = 0;
+	    math::PtEtaPhiMLorentzVector p4;
+	    
+	    if (useStage2L1_) {
```
siamo gia' dentro un `if(useStage2L1_)`, no? (L162)

battibass replied:
Ho aggiunto un paio di graffe e re indentato, ma non credo che fosse gia' dentro l'if.
Solo avevo l'indentazione automatica settata diversamente dalla tua quando ho scritto quel pezzo.

gpetruc commented on MuonAnalysis/MuonAssociators/interface/L1MuonMatcherAlgo.h:
```diff
@@ -133,16 +173,21 @@ class L1MuonMatcherAlgo {
+
+        typedef StringCutObjectSelector<l1t::Muon> L1TSelector;
+        /// Preselection cut to apply to L1 candidates before matching
+        L1TSelector preselectionCutL1T_;
```
Secodo me sia `l1extra::L1MuonParticle` e `l1t::Muon` ereditano  da `reco::Candidate`, quindi potrebbe bastare definirla come `StringCutObjectSelector<reco::Candidate,true>` e dovrebbe funzionare su entrambi con meno if e switch

battibass replied:
Ci provo ma ho una domanda stupida, se faccio come suggerisci poi il selettore accede correttamente ai membri che ci sono solo nelle classi derivate tipo "bx" o vede solo l'interfaccia di reco::Candidate ?

gpetruc replied:
Se ci metti il "true" come secondo parametro del template (o del
costruttore), lui vede tutti i metodi anche delle classi derivate.

On Fri, May 13, 2016 at 12:33 PM, Carlo Battilana notifications@github.com
wrote:

> In MuonAnalysis/MuonAssociators/interface/L1MuonMatcherAlgo.h
> https://github.com/gpetruc/cmssw/pull/8#discussion_r63163535:
> 
> > ```
> >      typedef StringCutObjectSelector<l1extra::L1MuonParticle> L1Selector;
> >      /// Preselection cut to apply to L1 candidates before matching
> > ```
> > -        L1Selector preselectionCut_;
> > -        L1Selector preselectionCutL1_;
> >   +
> > -        typedef StringCutObjectSelectorl1t::Muon L1TSelector;
> > -        /// Preselection cut to apply to L1 candidates before matching
> > -        L1TSelector preselectionCutL1T_;
> 
> Ci provo ma ho una domanda stupida, se faccio come suggerisci poi il
> selettore accede correttamente ai membri che ci sono solo nelle classi
> derivate tipo "bx" o vede solo l'interfaccia di reco::Candidate ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/gpetruc/cmssw/pull/8/files/1e8d982ddd89bac6791aaaf20485ada2a3d0b675#r63163535

battibass replied:
OK fatto e testato (scusa ci ho messo una vita).
Adesso preparo una PR che include i cambi al pacchetto di TnP per matchare al nuovo L1basata sui tuoi ultimi cambi nel repository ufficiale (finora, per la produzione di ntuple, abbiam usato questo branch + https://github.com/battibass/MuonAnalysis-TagAndProbe/tree/80X_trigger).
(Hugues ti dovrebbe rispondere alla mail sulla produzione di ntuple nel week end).

rekovic committed:
CaloL2 minbias, resolve conflicts for dev branch vs in release.

Commit:
Added a few suggestions from Giovanni

mulhearn committed:
remove unneeded casts

Commit:
use sys.stderr.write instead of print

Dr15Jones committed:
Added Erf and Landau functions to FormulaEvaluator

Added Erf and Landau functions which are needed for jet energy corrections.

perrotta committed:
Merged 80XMay10HLT from repository cms-tsg-storm

perrotta committed:
Update HLT menu up to /dev/CMSSW_8_0_0/HLT/V238

perrotta committed:
Update HLT menu up to /dev/CMSSW_8_0_0/HLT/V238

alja committed:
Fix bug in finding detail view for the class type.

alja committed:
Open detail view for aod data type.

alja committed:
enable gamma

alja committed:
disable debug prints.

alja committed:
configuration : save and restore palette.

fwyzard committed:
L1TGlobalPrescaler: implement arbitrary prescales on top of the existing L1 uGT results

sarafiorendi committed:
add tentative propagation to ME2 if MB2 fails

abrinke1 committed:
New structure to EMTF classes for unpacker, emulator, and DQM

abrinke1 committed:
Updated EMTF sub-addresses in RegionalMuonCand to match FW

abrinke1 committed:
Unpacker edits to suppress spurious EMTF warning messages

abrinke1 committed:
Missed two files on EMTF warning message commit

abrinke1 committed:
EMTF unpacker to match latest FW, use new classes

abrinke1 committed:
Neighbor sectors from https://github.com/mrcarver/cmssw/tree/NeighboringSectors, commit dc52b00

abrinke1 committed:
EMTF emulator outputs new class types

abrinke1 committed:
Proper translation of EMTF eta/phi to GMT scale

abrinke1 committed:
Minor EMTF DQM edits to not crash with new classes

abrinke1 committed:
Restore Mulhearn EMTF pT assignment - may not be up-to-date

ggovi committed:
Fix for GTEntry_t hash function, to allow to use the same tag for more record+labels in a GT

abrinke1 committed:
Small EMTF fixes for segfault, theta assignment, and pT calculation

usarica committed:
Zmumu tool mu- vs mu+ fixes

alja committed:
Comment EveUtil color settings.

alja committed:
Change setting for reco::Jet type.

eparadas committed:
Update BMTFUnpackerInputs.cc

Compatibility with older runs

eparadas committed:
Update BMTFUnpackerOutput.cc

Compatibility with older runs

rekovic committed:
Revert "Revert "- OMTF: fix eta assignement to (almost) realistic map""

This reverts commit 50b76210a3969cf0362fbe5bd3afcd2245db56a2.

kpedro88 committed:
fix more issues raised by Slava

fwyzard committed:
edmWriteConfig: fix python for large vectors

Python does not support function calls with more than 255 arguments:

    f(arg1, arg2, ..., arg255, ...)

The workaround is to wrap them in a tuple and expand it in place:

    f(*(arg1, arg2, ..., arg255, ...))

Add a check to `ParameterDescription.cc` so that `edmWriteConfig` can use
the latter syntax when generating `cfi.py` files with large vectors.

smorovic committed:
change defaults to empty-LS mode and micromerge by hltd (already switched on by default in CDAQ for some time)

smorovic committed:
enums for merge types. Values are defined in static const vector<string> member of EvFDaqDirector

cmsbuild committed:
Merge pull request #14521 from alja/pallete80

80x Fireworks: Implement palettes

cmsbuild committed:
Merge pull request #14499 from alja/detailView80

80x Fireworks:Fix problem in finding detail view for given class type

cmsbuild committed:
Merge pull request #14486 from alja/config80-2

 80x Fireworks:Update table view content (attempt No2)

smorovic committed:
fillJson has a new parameter. Use ROOT for mergeType

rmanzoni committed:
allow complete relax of tau isolation at high pt

smorovic committed:
added two JSON fields to DQMFileSaverPB

smorovic committed:
fixing bad syntax

rmanzoni committed:
add new parameter to the corresponding cfi

rmanzoni committed:
fix bool --> double

rekovic committed:
Fix L1REPACK:Full2015Data crash by removing Geometry

fioriNTU committed:
retune max value for Chi2

fioriNTU committed:
retuning also Pixel top Summary

davidlange6 committed:
Merge pull request #14502 from fwyzard/L1TGlobalPrescaler

L1TGlobalPrescaler: implement arbitrary prescales on top of the existing L1 uGT results

cmsbuild committed:
Merge pull request #14342 from hengne/from-CMSSW_8_0_7_relval

relval update data workflow reco step, and re-work the LHE-based fullSim workflows for HLT validation

alja committed:
New implementation of color randomization.

alja committed:
Add tooltip to randomize color buttons.

Commit:
Disable broken Ewk*DQM modules
(cherry picked from commit 45ddb62e5fc5db5ad14537de2802395d06d24c69)

rekovic committed:
Add L1REPACK:uGT to re-emulated only uGT using unpacked uGMT and Calo outputs

jmmans committed:
Fix swapped value1/value3

mulhearn committed:
cleanup of EMTF PH/TH LUTs requested by David L

alja committed:
Get access to singelton.

alja committed:
Change pallete order.

alja committed:
Add temproary shortcut to palettes.

alja committed:
Synchronize palette status from main frame and cmsShow popup.

davidlange6 committed:
Merge pull request #14522 from Dr15Jones/addMoreFunctionsToFormulaEvaluator_8_0

Added Erf and Landau functions to FormulaEvaluator

cmsbuild committed:
Merge pull request #14558 from alja/post-palette

80x Fireworks: Make shortcut to palette interface

cmsbuild committed:
Merge pull request #14506 from sarafiorendi/L2MuPropagateME2Fb_80X

add tentative propagation to ME2 if MB2 fails (backport of #14505)

mulhearn committed:
revert instance labels in BMTF unpacker which break re-emulation

OlivierBondu committed:
Update ResonanceDecayFilterHook.cc

mulhearn committed:
purge EMTF docs from repo

mulhearn committed:
Merged refs/pull/14362/head from repository cms-sw

mulhearn committed:
Merged refs/pull/14493/head from repository cms-sw

mulhearn committed:
Merged refs/pull/14419/head from repository cms-sw

Martin-Grunewald committed:
Merged 80XMay10HLT from repository cms-tsg-storm

idebruyn committed:
turn on ringView for TH1ClusterCharge and TH1ClusterStoNCorr

Martin-Grunewald committed:
Integration of CMSHLT-825 and preparation for CMSHLT-798

Martin-Grunewald committed:
CMSHLT 830 and 829

mulhearn committed:
clean-up L1T unpacker warnings

Commit:
Fix deltaR calculation in widejet module

Commit:
reco::MET --> reco::PFMET

konec committed:
assigne eta=+-121 to CSC hits with keyWG above |eta|=1.3, then filter them
set q=4 to muons with |eta|=115,
remove layers 7,8,9 from list of layers with q-4
minor fixes to CSC keywire group assignment to eta code

davidlange6 committed:
Merge pull request #14567 from dmitrijus/dqm_ewk_80

Disable broken Ewk*DQM modules

davidlange6 committed:
Merge pull request #14587 from duanders/widejetfix80X

Fix deltaR calculation in HLT widejet module

davidlange6 committed:
Merge pull request #14530 from rmanzoni/CMSSW_8_0_X

[RecoTauTag] allow complete relax of tau isolation at high pt

davidlange6 committed:
Merge pull request #14531 from fioriNTU/FixTrackingTopSummary80X

retune max value for Chi2

davidlange6 committed:
Merge pull request #14528 from smorovic/mergeType-80X

Adding merging type (DAQ) and changing defaults

davidlange6 committed:
Merge pull request #14566 from OlivierBondu/patch-3

Update ResonanceDecayFilterHook.cc

davidlange6 committed:
Merge pull request #14578 from CTPPS/ctpps_detid_backport

Adding new detectors - integration of RomanPot detectors for CTPPS project - backport of PR #13766

konec committed:
slightly tune/modify condition of q=4 for |eta|==115.
Now it is done if in case |eta|==115 and hits are ONLY in 3 layers out of (7,8,9,17)

konec committed:
In case of |eta| 121 set its quality to 4.
It has no effect for MC shince hits with this quality are filtered out, but
is usefull for data-MC comparistion (if one remove the filter)

kpedro88 committed:
fix else if in unpacker

ndaci committed:
Add a parameter: cut on the total charged energy fraction in the PFJet (maxCF). Default value: 99.

Martin-Grunewald committed:
L1T menu v3 and HLT development

cmsbuild committed:
Merge pull request #14588 from duanders/fixpfscoutingmet80X

Use PFMETCollection in scouting PF producer

cmsbuild committed:
Merge pull request #14501 from bartosik-hep/CMSSW_8_0_X_HLT_DQM

80X: HLT path/module-name updates in the HLTObjectMonitor

cmsbuild committed:
Merge pull request #14265 from mmusich/DropboxMetadata2016_80X

DropBox metadata - 2016 (80X)

ndaci committed:
Calculate std::abs(eta) only once.

davidlange6 committed:
Merge pull request #14570 from cms-l1t-offline/pr-l1t-tsg-v7-cand-80x

Update L1T to l1t-tsg-v7-cand performance (80x)

davidlange6 committed:
Merge pull request #14551 from cms-l1t-offline/rekovic-L1REPACK-2016-2015_CMSSW808

L1 repack 2016 2015 (Full, uGT, Full2015Data) cmssw808

davidlange6 committed:
Merge pull request #14477 from fwyzard/L1TGlobalProducer_migrate_to_stream_module

L1TGlobalProducer: migrate to stream module (80x)

Martin-Grunewald committed:
More HLT development / CMSHLT-818

cmsbuild committed:
Merge pull request #14580 from CTPPS/ctpps_rp_reco_data_formats_backport

CTPPS: new DataFormats for local TOTEM RP reconstruction, backport of #14134

cmsbuild committed:
Merge pull request #14467 from VinInn/FixCosmics80

fix det search tollerances for Cosmics

davidlange6 committed:
Merge pull request #13813 from vkhristenko/hcaldqm_newdev_v1_3appstest

Online HCAL DQM adding a 3rd application

davidlange6 committed:
Merge pull request #14512 from usarica/MuonAnalysisUpdates_8_0_X

Muon analysis updates 8.0.x

Martin-Grunewald committed:
More HLT from May 18th train

cmsbuild committed:
Merge pull request #14617 from cms-l1t-offline/pr-unpacker-warning-cleanup-80x

Clean-up L1T unpacker warnings

cmsbuild committed:
Merge pull request #14613 from cms-l1t-offline/pr-omtf-fix-80x

Update OMTF treatment of Muons at edge of OMTF acceptance

cmsbuild committed:
Merge pull request #14602 from ndaci/PR_80X_HLTPFJetID

 Add a cut in the HLTPFJetIDProducer module.

Martin-Grunewald committed:
Move to L1T menu v4

Martin-Grunewald committed:
Move to L1T menu v4 - take-2

mmusich committed:
Update the Global tags for latest L1 uGT menu

davidlange6 committed:
Merge pull request #14596 from smuzaffar/from-CMSSW_8_0_X_2016-05-20-2300

Fix for runtestTqafTopEventSelection unit test: use againstElectronVLooseMVA6 instead of againstElectronVLooseMVA5

cmsbuild committed:
Merge pull request #14579 from CTPPS/ctpps_raw_data_formats_backport

CTPPS: new DataFormats related to TOTEM RP raw data and digi - backport of PR #13837

Martin-Grunewald committed:
Use of new L1T v4 seeds

Martin-Grunewald committed:
CMSHLT 832 and 836 added to GRun

rekovic committed:
Adding L1REPACK FullMC to run on RAW of MC. Fix Csc tag for Omtf emulator in REPACK:Full, and clean REPACK:uGT

cmsbuild committed:
Merge pull request #14421 from diguida/fix_RunInfo_and_PopCon_80X

Fixes for RunInfo O2O and cleanup of PopCon interface (80X)

davidlange6 committed:
Merge pull request #14583 from ggovi/condcore-conddb-fix-for-gt-entry-hash-80X

Fix for GTEntry_t hash function

Martin-Grunewald committed:
CMSHLT-827 in GRun

rekovic committed:
Use default configuration with DTDigi_Source for simBmtfDigis

rekovic committed:
Change default caloParams to use Tau Iso Option 21 (caloStage2Params_2016_v2_2_cfi)

mulhearn committed:
restore overflow check for et and ht

cmsbuild committed:
Merge pull request #14437 from ggovi/condcore-utilities-o2o-1-80X

Changes for O2O deployment and monitoring

cmsbuild committed:
Merge pull request #13972 from fioriNTU/TkMapImprovements

improve TkMaps

davidlange6 committed:
Merge pull request #14647 from cms-l1t-offline/rekovic-L1REPACK-2016-2015_CMSSW808

Adding L1REPACK:FullMC workflow to run on RAW of MC (80x)

cmsbuild committed:
Merge pull request #14462 from kpedro88/uMNio-80X

Monitor CASTOR table data (80X)

cmsbuild committed:
Merge pull request #14650 from cms-l1t-offline/pr-l1t-sum-overflow

Restore overflow check for et and ht sums

davidlange6 committed:
Merge pull request #14646 from cms-l1t-offline/pr-mcprod-l1t-tauIsoOpt-21-80x

MCprod l1t Tau Iso Option-21 80x

ggovi committed:
Fix for handling copy of offline synchronized tags

VinInn committed:
fix crash in case Limit on the number of clusters is really exceeded.

davidlange6 committed:
Merge pull request #14600 from CTPPS/ctpps_raw_cond_formats_backport

CTPPS: new CondFormats related to TOTEM RP raw data and digi - backport of #13838

davidlange6 committed:
Merge pull request #14445 from cms-tsg-storm/80XMay10HLT

HLT development in May (80X)

cmsbuild committed:
Merge pull request #14662 from VinInn/FixTooManyClustersInSiPixelClusterProducer80

fix crash in case Limit on the number of clusters is really exceeded.

slava77devel committed:
cleanup RawToDigi redefinition of already loaded from RawToDigi_cff

davidlange6 committed:
Merge pull request #14568 from idebruyn/ringView_80X

TH1ClusterCharge and TH1ClusterStoNCorr plots per ring

davidlange6 committed:
Merge pull request #14658 from ggovi/conddb-utilities-conddb-tool-fix0-80X

Fix for handling copy of offline synchronized tags

davidlange6 committed:
Merge pull request #14336 from Martin-Grunewald/Stage2Prescales80X

Prescale access for stage-2 environment (80X)

davidlange6 committed:
Merge pull request #14677 from slava77/CMSSW_8_0_8_patch1/sign716/cleanupRawToDigi_Data

cleanup RawToDigi redefinition of already loaded from RawToDigi_cff (bp of #14675)

Commit:
Revert back to a single StringCutObjectSelector (StringCutObjectSelector<reco::Candidate,true>)

Commit:
Merged stage2_L1_matching from repository battibass

Commit:
Comment on the need ot dPhi offset

Commit:
remove import
